e2e/testcases/otel_collector_test.go

```diff
@@ -137,6 +146,13 @@ func TestOtelCollectorDeployment(t *testing.T) {
 	if err != nil {
 		nt.T.Fatal(err)
 	}
+
+	nt.T.Log("Test done, removing test resources")
```

Comment:
If this is just being done to remove test resources then this isn't necessary, the test scaffolding should handle cleanup of anything declared in the git repo

Reply:
Done!